core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/IntelliJProvider.scala

```diff
@@ -148,8 +148,14 @@ final case class IntelliJFactory(
     println(s"Installing $version")
     val file = dependencies.intelliJ.fetch(version)
     file.toExtracted.installTo(root)
-    root.resolve("bin").makeExecutableRecursively()
+    if (installedFromOsxDmg(root))
+      root.resolve("MacOS/idea").makeExecutable()
```

Comment:
Isn't MacOS dir present also in portable .zip distribution?

Reply:
You are right. It showed up that this change is not needed in fact, as `.makeExecutable()` on `MacOs/idea` is done inside `InstalledIntelliJ`

Reply:
I deleted this change from PR